main.css

```diff
@@ -0,0 +1,103 @@
+@import url('https://fonts.googleapis.com/css2?family=Roboto:wght@300;400;500;700&display=swap');
+
+:root{
+    --primary-color :#F65314;
+    --secondary-color :black;
+    --tertiary-color:#8da9bb;
+    --dark-color :#E50914; 
+    --light-color: white;
+    --yellow-shade-color :#FF9900;
+    --yellow-shade-color2: #ff7b25;
+    --green-shade-color1:#7CBB00;   
+    --green-shade-color2 :darkgreen;
+    --blue-shade-color1 :#4285F4;
+    --blue-shade-color2 : #003399;
+    --pink-shade-color : rgb(231, 64, 92) ;
+    --red-shade-color: #f7786b;
+    --cream-shade-color: #ffef96;
+    --cream-shade-color2:#8b6f47;
+    --button-primary-color1:#e0876a;
+    --button-primary-color2:#f4a688;
+    --button-floatHover-color:#292F33;
+    
+}
+*{
+    margin:0;
+    padding:0;
+}
+html{
+    font-family:Roboto, sans-serif;
+    /* font-size: 62.5%; */
+}
+a{
+    text-decoration:none;
+    color: var(--secondary-color);
+    
+}
+a:hover{
+    transform:scale(1.1);
+    color: var(--primary-color);
+}
+.link-active{
+    color: var(--primary-color);
+    /* text-decoration: underline; */
+}
+.fa-github-alt{
+    font-size: 25px;
+}
+.sidebar-docs-container{
+    display: flex;
+    justify-content: space-between;
+    /* border: solid red; */
+}
+.side-bar{
+    padding:0 30px ;
+    flex: 1;
+}
+.documentation{
+    flex-grow:3;
+    padding:0 20px 0 30px;
+    border-left: solid var(--primary-color);
+    height:80vh;
+}
+.p-utility{
```

Comment:
the naming convention can be improved here, bit difficult to understand what the purpose of .p-utility class